source/1_engine/rex_engine/include/rex_engine/diagnostics/logging/internal/details/formatting/flag_formatter.h

```diff
@@ -0,0 +1,936 @@
+#pragma once
+
+#include "rex_engine/diagnostics/logging/internal/details/formatting/padding_info.h"
+#include "rex_engine/diagnostics/logging/internal/details/log_msg.h"
+#include "rex_engine/diagnostics/logging/internal/details/os.h"
+#include "rex_std/array.h"
+
+namespace rexlog
+{
```

Comment:
naming code guidelines aren't followed in this file.
some make sense not to follow (eg. a_formatter) but I'd recommend to use them on the members inside these classes